app/models/concerns/inventory_computer.rb

```diff
@@ -29,6 +29,18 @@ def location_quantity(location)
         quantity_where(location: location)
       end
 
+      def total_quantity_in_box(box)
+        historical_quantity
+          .where(source: box, action: ['pack', 'unpack'])
+          .as_of_now
+      end
+
+      def quantity_of_package_in_box(package:)
```

Comment:
I feel like this doesn't make a lot of sense unless we specify which box to look into.
This will return the quantity in multiple boxes.

Reply:
ahh, good catch @patrixr . Will work on fixing this method.